openscap_report/report_generators/html_templates/evaluation_characteristics.html

```diff
@@ -155,6 +155,20 @@ <h3>
                         </div>
                     </div>
                 </li>
+
+                {%- for type, addresses in report.scan_result.target_addresses.items() -%}
+                <li class="pf-c-data-list__item">
+                    <div class="pf-c-data-list__item-row">
+                        <div class="pf-c-data-list__item-content">
+                            <div class="pf-c-data-list__cell"><b>Target {{ type }} addresses:</b></div>
```

Comment:
It would be better if you used the correct orthography of these acronyms, ie. MAC, IPv4, IPv6 instead of mac, ipv4, ipv6.

Reply:
Ok, here is the fixed version.

![image](https://github.com/OpenSCAP/openscap-report/assets/26072444/25f48f54-6c83-4ad5-b3ee-f35c08b9b632)